### PR TITLE
Rocket launch 15 create text components

### PR DESCRIPTION
**Pull request by moustafaelsisy**
* Defined text styles in index.css
* Deleted App.css
* Removed default code in App.js
* Added our own demo code inside App.js, to serve as a guide on how to use our common components
* Added MuiThemeProvider to App.js, which is necessary for material-ui to work

*To test, run `npm run start`*

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,19 +1,36 @@
...
-          <h2>Welcome to React</h2>
-        </div>
-        <p className="App-intro">
-          To get started, edit <code>src/App.js</code> and save to reload.
-        </p>
+    <MuiThemeProvider>
+      <div>
+        <DemoTile color="white" backgroundColor="navy">
+          <CenterDiv>
+            <h1>St. John's<br/>AI Hackathon<br/>2017</h1>
```
Comment (moustafaelsisy) [inline frontend/src/App.js]: Doesnt seem like GitHub understands JSX perfectly? o.o

File: frontend/src/App.js [hunk 2]
```diff
@@ -1,19 +1,36 @@
...
+            <h2>Powered by</h2>
+            <h3>MUN CS Society</h3>
```
Comment (ToJen) [inline frontend/src/App.js]: Should we do something like:
```html
<h2>Powered by 
    <a href="https://muncompsci.ca">
        <img src="https://muncompsci.ca/static/logo.png" style="whatever_looks_good"/>
    </a>
</h2>
```

P.S: Didn't crosscheck my code for accuracy there

Comment (ToJen) [inline frontend/src/App.js]: View [here](https://ibb.co/gWTbZv)

Comment (moustafaelsisy) [inline frontend/src/App.js]: Oh boy! That MUN CS logo looks great! Though the code I have in App.js is just demo code to show how the font styles look like and is not the actual title tile 😂 Though keep that logo for the actual title tile task! :D

Comment (ToJen) [inline frontend/src/App.js]: lol ok, we'll have to ask for a more landscape one as well

Comment (moustafaelsisy) [inline frontend/src/App.js]: Ya thats what I thought as well

**Pull request merged**